### ISSUE TITLE
Syslog option (-y) doesn't work

### ISSUE THREAD

**Issue by davidmatson** [PowerShell/Win32-OpenSSH#1684]
**"OpenSSH for Windows" version**
7.7.2.3
 
**Server OperatingSystem**
Windows 10 Enterprise 

**Client OperatingSystem**
Windows 10 Enterprise 

**What is failing**
-y option of ssh.exe does nothing (even with -vvv)

**Expected output**
Messages sent to syslog (localhost UDP port 514)

**Actual output**
No messages sent to syslog

**Details**
I've tried running multiple local syslog servers and confirmed they can see syslog messages sent from other clients. But when I use ssh.exe with the -y option, no syslog messages get sent, even at the most verbose level (-vvv). Is there a known gap here?

**Comment by WSLUser** [comment]
This won't fix it likely but can you verify you hit this with 8.1? Release is here on the github

**Comment by davidmatson** [comment]
Yes, I tried the [latest release](https://github.com/PowerShell/Win32-OpenSSH/releases/tag/v8.1.0.0p1-Beta) and didn't see any syslog output from it either.
